### DISCUSSION TITLE
How does BLE's central and peripheral synchronize the frequency hopping frequency

### DISCUSSION THREAD

**Discussion by mfinmuch** [zephyrproject-rtos/zephyr#29184] [Q&A]
Currently I use nrf52840 for BLE research
I know that in bluetooth/controller/ll_sw/nordic/lll/lll_master.c, in prepare_cb, data_chan_use is setting the frequency channel to be used next
`data_chan_use = lll_chan_sel_2(event_counter, lll->data_chan_id,
					       &lll->data_chan_map[0],
					       lll->data_chan_count);`
But how does central inform the frequency sequence to the peripheral to jump?
Can't find a place for the synchronization frequency hopping sequence
Please tell me if you know
thank

**Comment by carlescufi** [comment]
@cvinayak can probably answer that

**Reply by mfinmuch** [reply]
@carlescufi thanks,
How can i contact him?

**Reply by carlescufi** [reply]
He's around in Zephyr's Slack, with the same handle.
